tauri-app/src-tauri/src/wine_manager.rs

```diff
@@ -55,6 +53,19 @@ impl WineManager {
         .spawn().expect("Failed to init prefix").wait().expect("Failed to init prefix");
     }
 
+    pub fn launch_exe(&self, absolute_path: &str, args: &Vec<&str>) -> std::io::Result<()> {
+        let display = env::var("DISPLAY").unwrap_or_else(|_| ":0".to_string());
```

Comment:
I'm not sure defaulting the value for `DISPLAY` is the right approach. Have you seen this happen?

Likewise for Wayland, you also have WAYLAND_DISPLAY which may need factoring in here - from my own system:

```DISPLAY=:1
WAYLAND_DISPLAY=wayland-0```

Reply:
Wine runs in xwayland by default so there is no need to set the `WAYLAND_DISPLAY` variable. I'm unsure what best practice is as far as setting the `DISPLAY` when it isn't set on the host, but if it is unset the program will fail to launch.